apps/web/src/services/asset-inventory/collector/modules/CollectorContents.vue

```diff
@@ -117,25 +121,33 @@ const handlerState = reactive({
         created_at: makeDistinctValueHandler('inventory.Collector', 'created_at'),
         last_collected_at: makeDistinctValueHandler('inventory.Collector', 'last_collected_at'),
     } as ValueHandlerMap,
+    excelFields: [
+        { key: 'name', name: 'Name' },
+        { key: 'state', name: 'State' },
+        { key: 'plugin_info.plugin_id', name: 'Plugin' },
+        { key: 'plugin_info.version', name: 'Version' },
+        { key: 'last_collected_at', name: 'Last Collected', type: 'datetime' },
+    ],
 });
 
-const state = reactive({
-    selectedProvider: computed(() => cloudCollectorPageState.selectedProvider),
-});
 
-const emit = defineEmits(['export-excel', 'change-toolbox']);
+const emit = defineEmits(['change-toolbox']);
 
 /* Components */
 const handleSelectedProvider = (providerName: string) => {
-    cloudCollectorPageStore.setSelectedProvider(providerName);
+    collectorPageStore.setSelectedProvider(providerName);
 };
 const handleCreate = () => {
     SpaceRouter.router.push({ name: ASSET_INVENTORY_ROUTE.COLLECTOR.CREATE._NAME });
 };
-const handleExport = async () => {
-    emit('export-excel');
-};
+const handleExport = async () => {};
 const handleChange = async (options) => {
+    const searchQueryHelper = new QueryHelper();
```

Comment:
nitpick: This could be placed outside the function! Alternatively, `QueryHelper()` can be created each time the function is executed.